app/routes/_extras.brand.tsx

```diff
@@ -0,0 +1,345 @@
+        use of the Remix name or logos in a manner that could cause customer
+        confusion is not permitted.
+      </p>
+      <p>
+        Additionally, you may not use our trademarks for t-shirts, stickers, or
+        other merchandise without explicit written consent.
+      </p>
+      <AssetHeader>Logo</AssetHeader>
+      <p>
+        Please use the logo with appropriate background. On dark backgrounds use
+        the light or glowing logo, and on light backgrounds use the dark logo. You may also{" "}
+        <a
+          content="Google drive containing all image files for Remix Logo\'s"
+          href="https://drive.google.com/drive/u/0/folders/1pbHnJqg8Y1ATs0Oi8gARH7wccJGv4I2c"
+          className="text-blue-brand hover:underline"
+        >
+          {" "}
+          download all image files for Remix Logo's in bulk through Google
+          Drive.
+        </a>
+      </p>
+      <LogoBox name="remix-letter" Logo={LogoRemixLetter} />
+      <AssetHeader>Logo Word</AssetHeader>
+      <LogoBox name="remix" Logo={LogoRemix} />
+    </div>
+  );
+}
+
+function AssetHeader({ children }: { children: React.ReactNode }) {
+  return (
+    <h2 className="text-xl font-extrabold dark:text-gray-200 md:text-3xl">
+      {children}
+    </h2>
+  );
+}
+
+interface LogoProps {
+  fg: string;
+  highlight: string;
+  outline?: string;
+}
+
+function LogoBox({
+  name,
+  Logo,
+}: {
+  name: "remix-letter" | "remix";
+  Logo: FC<LogoProps>;
+}) {
+  // Tailwind classnames for the various ways we style the individual
+  // component pieces of each LogoBox variant
+  let variants = {
+    light: {
+      bg: "bg-white",
+      border: "border-gray-50 dark:border-transparent",
+      fg: "fill-gray-900",
+      highlight: "fill-black-brand",
+      outline: "black",
+    },
+    "light-outline": {
+      bg: "bg-white",
+      border: "border-gray-50 dark:border-transparent",
+      fg: "fill-gray-900",
+      highlight: "fill-transparent",
+      outline: "black",
+    },
+    dark: {
+      bg: "bg-gray-900",
+      border: "border-transparent dark:border-gray-800",
+      fg: "fill-gray-900",
+      highlight: "fill-white",
+      outline: "white",
+    },
+    "dark-outline": {
+      bg: "bg-gray-900",
+      border: "border-transparent dark:border-gray-800",
+      fg: "fill-gray-900",
+      highlight: "fill-transparent",
+      outline: "white",
+    },
+    "glowing": {
+      bg: "bg-gray-900",
+      border: "border-gray-50 dark:border-transparent",
+      fg: "fill-gray-900",
+      highlight: "fill-white",
+      outline: "glow",
+    },
+  };
+  return (
+    <div className="grid grid-cols-2 grid-rows-2 gap-4 gap-x-6">
+      {Object.entries(variants).map(
+        ([variant, { bg, border, fg, highlight, outline }]) => (
+          <div className="flex flex-col" key={variant}>
+            <div
+              className={`flex h-40 items-center justify-center rounded-lg border-[3px] p-4 md:h-48 ${bg} ${border}`}
```

Comment:
Totally not a big deal, but we usually use the `cx` utility (`import cx from "clsx";`) for this sort of pattern. Just makes it a little easier to combine a bunch of classnames

```suggestion
              className={cx(
                "flex h-40 items-center justify-center rounded-lg border-[3px] p-4 md:h-48",
                bg,
                border,
              )}
```

Reply:
Ah cool! Thanks for pointing this out.